docs/_docs/test.md

```diff
@@ -3,153 +3,120 @@ nav_order: 6
-| ``-V``   | ``--mqttVersion``| The MQTT version to test the broker on. | Both versions will be tested
-| ``-a``| ``--all`` | Perform all tests for all MQTT versions | ``false`` (Only test MQTT 3)
-| ``-t``| ``--timeOut`` | The time to wait for the broker to respond (in seconds). | ``10s``
-| ``-q`` | ``--qosTries`` | The amount of messages to send and receive from the broker for each QoS level. | ``10``
-| ``-l`` | | Log to ~./mqtt.cli/logs (Configurable through ~/.mqtt-cli/config.properties) | ``false``
+#### Credentials Authentication
 
+{% include options/authentication-options.md %}
 
-***
+#### TLS Authentication
 
-## Connect Options
+{% include options/tls-options.md %}
 
-|Option   |Long Version    | Explanation                                         | Default|
-|---------|----------------|-----------------------------------------------------|---------|
-| ``-h``   | ``--host``| The MQTT host. | ``localhost``
-| ``-p``  | ``--port``| The MQTT port. | ``1883``
+### Logging Options
 
-***
+{% include options/logging-options.md %}
 
-## Security Options
-
-|Option   |Long Version    | Explanation                                         | Default|
-|---------|----------------|-----------------------------------------------------|---------|
-| ``-s``    | ``--secure``  | Whether a default SSL configuration is used. | ``false``
-| ``-u``   | ``--user`` | Define the username for authentication. |
-| ``-pw``  | ``--password`` | Define the password for authentication directly. <br> If left blank the user will be prompted for the password in console. |
-| ``-pw:env``  |  | Define that the password for authentication is read in from an environment variable. | ``MQTT_CLI_PW`` if option is specified without value
-| ``-pw:file``  |  | Define the path to a file from which the password is read in. |
-|   |   ``--cert``  |   The path to the client certificate to use for client-side authentication. |
-|   |   ``--key``   |   The path to the corresponding private key for the given client certificate.    |
-|   | ``--cafile``    | The path to the file containing a trusted CA certificate to enable encrypted certificate based communication. |
-|   | ``--capath``  | The path to the directory containing trusted CA certificates to enable encrypted certificate based communication. |
-|   | ``--ciphers``  | The supported cipher suites in IANA string format concatenated by the ':' character if more than one cipher should be supported. <br> e.g ``TLS_CIPHER_1:TLS_CIPHER_2`` <br> See [IANA Format](https://www.iana.org/assignments/tls-parameters/tls-parameters.xml) for supported cipher suite strings. |
-|   |   ``--tls-version``   |   The TLS version to use - ``TLSv1.1`` ``TLSv1.2`` ``TLSv1.3`` | ``TLSv1.2`` |
+### Help Options
+
+{% include options/help-options.md defaultHelp=false %}
 
 *** 
 
 ## Further Examples
 
-> Perform tests for MQTT 5 only
+Perform tests for MQTT 5 only
 
 ```
 $ mqtt test -h broker.hivemq.com -a -V 5
```

Comment:
if `-a` means test all versions, doesn't adding -V 5 contradict this?

Reply:
The test is for all tests in the MQTT 5 scope. -a enables additional tests for MQTT 5. The description of -a is somewhat misleading.